### DISCUSSION TITLE
Guidelines for reduced/increased contrast modes?

### DISCUSSION THREAD

**Discussion by Seirdy** [Myndex/SAPC-APCA#82] [Q&A] [answered]
Given that L<sup>C</sup>=90 is the ideal body-text contrast for most color schemes: what are ideal targets for reduced and increased contrast modes?

On the Web, <cite>[CSS Media Queries Level 5](https://drafts.csswg.org/mediaqueries-5/#prefers-contrast)</cite> specifies that `less` and `more` are some of the possible values for the `prefers-contrast` media feature.

I specify custom colors for a dark theme (`prefers-color-scheme: dark`). Right now, I target a lightness contrast between -85 and -84 for `less`, and between -95 and -97 for `more`.

**Comment by Myndex** [comment, marked as answer]
Hi @Seirdy 

This specific subject is on the "future studies list" for a clinical study to help define these objectives.

If you'd accept an informed but not tested opinion, I'd suggest that "high" contrast modes ADD at least Lc 15 to all cases, and a "low" contrast mode SUBTRACT Lc 15 from all cases, _BUT never less than the "absolutes"_, i.e. for LOW contrast mode, no text at all under Lc 30, no fluent text less than Lc 45, no body text less than Lc 60.

### Procedure:
For **HIGH contrast mode**, then, In the APCA tool, use a minimum font weight and size for an Lc value that is 15 _LOWER_ than the colors you are using.

For **LOW contrast mode**, then, In the APCA tool, use a minimum font weight and size for an Lc value that is 15 _HIGHER_ than the colors you are using. But no text colors less than Lc 30, no fluent text less than Lc 45, and no body text less than Lc 60 as hard limits.


_**DISCLAIMER:**_ This opinion is based on current trials and experience, but has not otherwise been tested for veracity. Feedback and discussion here is encouraged.

**Reply by Seirdy** [reply]
> If you'd accept an informed but not tested opinion, I'd suggest that "high" contrast modes ADD at least Lc 15 to all cases, and a "low" contrast mode SUBTRACT Lc 15 from all cases, BUT never less than the "absolutes", i.e. for LOW contrast mode, no text at all under Lc 30, no fluent text less than Lc 45, no body text less than Lc 60.

I don't see how this could work for a dark theme. The current recommendation is to target Lc=-90. An additional -15 would put me at Lc=-105, which is close to the maximum contrast. My text would need to be nearly white and the background would be nearly (or completely) black. However, links need to be colored; they can't have the full `#fff`. To distinguish their color, at least one of the three primary colors will need to reduce. The result is that links will look noticeably dimmer than plain foreground text. The effect seems to be more pronounced for dark themes, where color limits the maximum brightness.

**Reply by Myndex** [reply]
Similar to locking the minimum contrasts, also lock the max at lc90.
